testing/ios/IosUnitTests/IosUnitTests.xcodeproj/project.pbxproj

```diff
@@ -446,6 +446,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = "";
```

Comment:
let's revert this.

Reply:
I'm referring to reverting this to the original state where the line `DEVELOPMENT_TEAM = "";` is not included.

Reply:
Got it. I didn't realize the whole line was added, I thought I just updated the string. I'll revert, thanks!